Classes/Issues/Branches/IssueTargetBranchCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  IssueTargetBranchCell.swift
+//  Freetime
+//
+//  Created by Yury Bogdanov on 13/03/2018.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+final class IssueTargetBranchCell: AttributedStringCell {
```

Comment:
Would love to land #1674 first and then merge this over to use the new text system!

Reply:
Ok then 🙂 Waiting for #1674 merge